### ISSUE TITLE
MISSING_VAR not triggering for tags in script

### ISSUE THREAD

**Issue by KamasamaK** [cflint/CFLint#574]
The following code should trigger a MISSING_VAR on `httpResponse`, but does not.

```
/**
 * Some function
 * @call The value of the call
 */
public void function foo(
	required string call
)
{
	cfhttp(url="#variables.host#", port="#variables.port#", method="GET", timeout="30", result="httpResponse") {
		cfhttpparam(type="url", name="call", value="#arguments.call#");
	}
}
```

It triggers fine on the standard tag equivalent.

**Comment by ryaneberly** [comment]
I implemented this in a "pseudo" fashion.  Jericho HTML Parsing in not extensible, so it is not trivial to combine both cfml and these cfscript tags as a single abstraction.  For now, CFLint generates cfml code for the cfscript 'tag'.  Seems to work well for basic cases like this.
